websites/J/Jackbox/presence.ts

```diff
@@ -49,84 +49,135 @@ const presence = new Presence({
 // [x] Drawful 2
 // [x] Quiplash
 
+let layout: string;
+const elapsed: number = Math.round((new Date()). getTime() / 1000);
+
 presence.on("UpdateData", async () => {
   const presenceData: presenceData = {
     largeImageKey: "jackbox"
   };
 
+  // Check for presence setting
+  const useName: boolean = await presence.getSetting("useName");
+  const useTime: boolean = await presence.getSetting("useTime");
+
+  if(useTime){
+    presenceData.startTimestamp = elapsed;
+  }
+
+  if(useName && layout == "new"){
+    presenceData.state = "as " + document.getElementById("playername").innerHTML;
+  }
+
+  if(useName && layout == "legacy"){
+    presenceData.state = "as " + document.getElementById("player").children[0].innerHTML;
+  }
+
+  if(useName && layout == "dict"){
+    presenceData.state = "as " + document.getElementById("playericon").className.split("_")[1] + document.getElementById("playername").innerHTML.toLowerCase();
+  }
```

Comment:
you should put these at the bottom since layout is not set yet.

Reply:
k